174962304 text overflow

mklewandowski opened the pull request:
This PR fixes two cases where long, overflowing text was not being handled properly.  In both cases overflow text should be shown with a set number of lines and with ellipsis as per the UI spec.  Cases include:
- activity names in the buttons in the level viewer
- activity name in the all student responses view
![image](https://user-images.githubusercontent.com/5126913/94226004-55c41d00-feab-11ea-8645-6b2464ec84d9.png)

For anyone interested, I used the library `react-lines-ellipsis` to handle multi-line overflow (which is much trickier than single line overflow with ellipsis).  I also tried `react-clamp-lines` (I couldn't get the sizing and number of clamped lines to work) and `React-clamp` (old, didn't work at all).

eireland reviewed (approved):
Looks good. I'm wondering if we should also implement this in the page titles 
![image](https://user-images.githubusercontent.com/7716653/94300345-b2f8b680-ff1d-11ea-9e0f-f44be292caed.png)

mklewandowski commented:
> Looks good. I'm wondering if we should also implement this in the page titles
> ![image](https://user-images.githubusercontent.com/7716653/94300345-b2f8b680-ff1d-11ea-9e0f-f44be292caed.png)

Ah, yes!  Good catch!  I added the ellipsis to the page name container.  I also added tooltip text so that when a user hovers, they can see the full text.

kswenson reviewed (commented):
👍
